### PR TITLE
mdd-task2

### PR DESCRIPTION
**Pull request by mddifilippo89**
I had to fix gulpfile.js with poly/polyline if checking

Note**
getting undefined in the SampleDisplayName

eg.
- component Spreadsheet
/excel/spreadsheet-config-options === undefined
/excel/spreadsheet-activation === undefined
/excel/spreadsheet-adapter-combo === Adapter Combo
/excel/spreadsheet-clipboard === undefined
/excel/spreadsheet-commands === undefined
/excel/spreadsheet-conditional-formatting === undefined
/excel/spreadsheet-adapter-chart === undefined
/excel/spreadsheet-data-validation === Data Validation
/excel/spreadsheet-filter-dialog === Filter Dialog
/excel/spreadsheet-format-dialog === Format Dialog
/excel/spreadsheet-hyperlinks === undefined
/excel/spreadsheet-overview === undefined
/excel/spreadsheet-sort-dialog === Sort Dialog

### CODE REVIEW

File: tasks/gulp-samples.js [hunk 1]
```diff
@@ -475,6 +475,13 @@ function logSourceFiles(cb) {
     gulp.src([
         './samples/**/src/*.ts',
        '!./samples/**/src/index.*',
+       '!./samples/**/src/*Utility.ts',
+       '!./samples/**/src/*Utils.ts',
+       '!./samples/**/src/*Sample.ts',
+       '!./samples/**/src/Products.ts',
+       '!./samples/**/src/PeriodicElements.ts',
+       '!./samples/**/src/WorldConnections.ts',
+       '!./samples/**/src/WorldLocations.ts',
```
Comment (HUSSAR-mtrela) [inline tasks/gulp-samples.js:484]: you need to list here all shared file names that are in the templates/shared/src folder

File: gulpfile.js [hunk 2]
```diff
@@ -56,10 +56,10 @@ let contains = require('gulp-contains');
 // exports.logRootFiles   = sb.logRootFiles;
 // exports.lintSamples    = sb.lintSamples;
 
-// exports.logRoutes = logRoutes = gulp.series(
-    // sb.getSamples,
-    // sb.logRoutes,
-// );
+exports.logRoutes = logRoutes = gulp.series(
+    sb.getSamples,
+    sb.logRoutes,
```
Comment (HUSSAR-mtrela) [inline gulpfile.js:61]: add logging of sample.SampleFileName in the same console.log() in sb.logRoutes()

File: tasks/gulp-samples.js (hunk 1 above)
Comment (mddifilippo89) [reply tasks/gulp-samples.js:513]: done

File: gulpfile.js (hunk 2 above)
Comment (mddifilippo89) [reply gulpfile.js:61]: Done

for (const sample of component.Samples) {
                console.log('' + sample.SampleRoute + '/' + sample.SampleFileName + ' === ' + sample.SampleDisplayName);
            }

File: tasks/gulp-samples.js (hunk 1 above)
Comment (mddifilippo89) [reply tasks/gulp-samples.js:513]: Done

**Pull request merged**